extensions/vscode/src/test/suite/commands/create.test.ts

```diff
@@ -45,7 +45,7 @@ suite("create command", () => {
 
   suite("file open dialog", () => {
     test("is shown when Uri is undefined", async () => {
-      vscodeStub.window.showOpenDialog.returns(Promise.resolve(undefined));
+      vscodeStub.window.showOpenDialog.resolves(undefined);
```

Comment:
Pass undefined is a little bit redundant as it is implicit

```suggestion
      vscodeStub.window.showOpenDialog.resolves();
```